draft-ietf-quic-http.md

```diff
@@ -1598,9 +1654,11 @@ extension is disabled if the setting is omitted.
 # Security Considerations
 
 The security considerations of HTTP/3 should be comparable to those of HTTP/2
-with TLS; the considerations from Section 10 of {{!HTTP2}} apply in addition to
+with TLS; the considerations from Section 10 of {{?HTTP2}} apply in addition to
 those listed here.
 
+TODO:  This is going to be a big import, probably worthy of its own PR.
```

Comment:
Leftover?  If not, I'm not sure what this TODO is exactly indicating.

Reply:
"worthy of its own PR" => #3407 addresses all imports *except* Security Considerations.

Reply:
@martinthomson notes that many of the Security Considerations of 7540 are from transporty stuff, and we might have less to import here than it seems.

Reply:
This wound up being pretty big.  It will follow as a separate PR to encourage appropriate review.